manifests/params.pp

```diff
@@ -9,6 +9,7 @@
   $venv_path           = '/opt/letsencrypt/.venv' # virtualenv path for vcs-installed letsencrypt
   $repo                = 'https://github.com/certbot/certbot.git'
   $cron_scripts_path   = "${facts['puppet_vardir']}/letsencrypt" # path for renewal scripts called by cron
+  $key_size            = 4096
```

Comment:
While I personally prefer 4096, AFAIK LE still defaults to 2048.

Reply:
Do we need to follow the same thing? Or shall the default be the higher?

Reply:
Not sure. @Dan33l thoughts?

Reply:
I believe it should be 4096. If people want less, they can override. Security should be default in my opinion.

Reply:
In general, i try to stick with managed product defaults.
But about security, if a more secure setting does not cause trouble, i am fine with this.

Reply:
So i agree with using default to 4096.
https://github.com/certbot/certbot/issues/489